packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/pos-block/default.example.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  POSBlock,
+  extension,
+} from '@shopify/ui-extensions/point-of-sale';
+
+export default extension(
+  'pos.purchase.post.block.render',
+  (root, api) => {
+    const posBlock = root.createComponent(
+      POSBlock,
+      {
+        action: {
+          title: 'A toast message',
+          onPress: () => {
+            api.Toast.show('A toast message');
+          },
+        },
+      },
+    );
+
+    root.append(posBlock);
```

Comment:
We probably want an example with the actual POSBlockRow being appended to this eh? Then we can use the single example for both components' docs?

Reply:
I don't like writing examples without verifying they work anyway, so I'd rather revisit when we have screenshots and working code so I can guarantee the quality of these. That's why I started barebones. Agreed that it can be the same example for both pages tho.